packages/popup/src/pages/CurrentSeed/index.js

```diff
@@ -70,53 +78,78 @@ function SeedPhrase({group}) {
       tabIndex="-1"
       key={group.eid}
       className="h-12 px-3 hover:bg-primary-4 flex items-center cursor-pointer justify-between"
-      onClick={() => setSelectedGroup(group)}
+      onClick={() => {
+        setSelectedGroup(group)
+        setAccountName(`Create-${idx + 1}-${group.account.length + 1}`)
+      }}
       onKeyDown={() => {}}
     >
       <div className="flex items-center">
         <span className="text-gray-80 mr-2">{nickname}</span>
         <span className="text-gray-40">{accountLength}</span>
       </div>
-      {group.eid === selectedGroup.eid && <Selected className="w-5 h-5" />}
+      {group.eid === selectedGroup?.eid && <Selected className="w-5 h-5" />}
     </div>
   )
 }
 
 SeedPhrase.propTypes = {
   group: PropTypes.object.isRequired,
+  idx: PropTypes.number,
 }
 
 function CurrentSeed() {
-  const {t, accountName, setAccountName, selectedGroup, hdGroup, onCreate} =
-    useStore()
+  const {
+    t,
+    accountName,
+    setAccountName,
+    setAccountNameError,
+    selectedGroup,
+    hdGroup,
+    accountNameError,
+    onCreate,
+  } = useStore()
 
   return (
-    <div className="w-full h-full px-4 pb-4 flex flex-col bg-bg items-center">
+    <div className="h-full px-3 flex flex-col bg-bg">
       <CompWithLabel label={t('accountName')}>
         <Input
           width="w-full"
           value={accountName}
-          onChange={e => setAccountName(e.target.value)}
+          onChange={e => {
+            setAccountName(e.target.value)
+            const accountNames = selectedGroup.account.map(a => a.nickname)
+            if (accountNames.indexOf(e.target.value) !== -1) {
+              setAccountNameError(t('errorDuplicateName'))
+            } else {
+              setAccountNameError(null)
+            }
```

Comment:
better put this in `setAccountName`

how about using reduce implement the duplicate check logic
```js
      selectedGroup.account.reduce(
        (noDup, {nickname}) => noDup && nickname !== e.target.value,
        true,
      ) || setAccountNameError(t('errorDuplicateName'))
```

Reply:
Done